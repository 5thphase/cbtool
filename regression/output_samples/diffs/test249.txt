|name |role |size |cloud_ip |host_name |vmc_pool |netname |ai |aidrs |uuid
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|Redis Version |2.8.6 
|HOSTs |60 
|VMCs ISSUED |16 
|HOSTs ISSUED |0 
|VMs ISSUED |22 
|VMs SLA PROVISIONING VIOLATED |0 
|VMs SLA RUNTIME VIOLATED |0 
|VMs APPLICATION ERRORS |0 
|AIs ISSUED |0 
|AIDRSs ISSUED |0 
|VMCRSs ISSUED |0 
|FIRSs ISSUED |0 
|EXPERIMENT COUNTER |234 
