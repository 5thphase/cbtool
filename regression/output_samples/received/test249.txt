 status: Sending a termination request for vm_1 (cloud-assigned uuid F20A2D6F-5B37-5CFD-8FC2-E21079C408EE)....
VM object 27F53EF9-C218-54C0-AE96-3122E861A23C (named "vm_1") was sucessfully detached from this experiment.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_3          |tinyvm              |pico32      |83.24.178.34    |simhostb1                  |SUT                 |none    |none       |B40E64FC-6FBC-58CB-A1C0-DC0CEE72D17A
|vm_6          |tinyvm              |pico32      |47.78.8.159     |simhostb0                  |SUT                 |none    |none       |05AAC1DB-7FD8-5032-B0D7-369DDF917E97
|vm_7          |tinyvm              |pico32      |166.31.113.1    |simhostb2                  |SUT                 |none    |none       |DFBB15E7-B73B-5FF5-B3E9-8AB0B72D8639
|vm_8          |tinyvm              |pico32      |12.103.98.234   |simhosta0                  |SUT                 |none    |none       |678C0AF7-0389-5E8D-82B9-DCE0F795D792
|vm_9          |db2                 |platinum64  |46.109.195.91   |simhostb3                  |SUT                 |none    |none       |4F780BED-D6CE-5508-8EFF-9020DCF3FF05
|vm_10         |netclient           |iron32      |83.2.183.19     |simhosta2                  |SUT                 |none    |none       |FC7F0411-83A8-5C4F-A8C1-34317D39F27C
|vm_11         |fen_hpc             |platinum64  |28.83.253.116   |simhostd4                  |LG                  |none    |none       |67B0289A-479C-53E3-A608-9D7A8AE7F5CA
|vm_12         |tinyvm              |iron32      |203.176.131.157 |simhostd0                  |SUT                 |none    |none       |1DEA68F7-630C-5485-9760-97A912604217
|vm_13         |tinyvm              |iron32      |181.51.23.51    |simhostb3                  |SUT                 |none    |none       |CDA22BF8-8F44-585D-A365-70AC86CE82EB
|vm_14         |predictablevm       |iron32      |1.2.3.4         |simhostc1                  |SUT                 |none    |none       |2E9AB0B4-CFF1-5A7B-9F1E-DB8EA44994A6
|vm_15         |tinyvm              |pico32      |215.163.241.67  |simhosta2                  |SUT                 |none    |none       |F6A64894-7D92-585C-B407-8A14ACB68E28
|vm_16         |tinyvm              |pico32      |145.36.198.108  |simhostd1                  |SUT                 |none    |none       |062922BC-6F2F-5AA0-9582-B9DD84CC32E1
|vm_17         |tinyvm              |pico32      |7.166.81.164    |simhosta0                  |SUT                 |none    |none       |D936D335-2217-5599-AE40-A928F6A94044
|vm_18         |tinyvm              |pico32      |235.165.50.238  |simhostb4                  |SUT                 |none    |none       |856D8533-B84A-5EA2-A5B5-40A7441E80DE
|vm_19         |tinyvm              |pico32      |229.69.116.115  |simhostc3                  |SUT                 |none    |none       |01C2FCC9-C96A-5747-9F13-CD495A15EE96
|vm_20         |tinyvm              |pico32      |97.69.98.201    |simhostd3                  |SUT                 |none    |none       |88ACF8BD-6990-5B71-82FD-A4C0E1B4D36B
|vm_21         |tinyvm              |pico32      |193.130.70.163  |simhostb2                  |SUT                 |none    |none       |A10E7991-11B7-5778-B51D-1A65234CF0AF
|vm_22         |tinyvm              |pico32      |170.41.160.105  |simhostd0                  |SUT                 |none    |none       |186CA1BC-C6CF-5D07-818E-9B24D198B309

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |2.78M                         
|Redis Version                                        |2.8.4                         
|Uptime (in seconds)                                  |3057781                       
|Total Connections Received                           |128134                        
|Total Commands Processed                             |9443383                       
|Number of Keys                                       |427                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |3399680                       
|Data Size                                            |170736                        
|Index Size                                           |294336                        
|Average Object Size                                  |405.548693587                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |18                            
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |18                            
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |4                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |114                           

 ---------------- VMC-WIDE COUNTERS ----------------
|99276BDE-868D-5D11-B20B-7B4C612399A6 (simzone_a) VM RESERVATIONS|6                             
|5C042653-2D38-5F10-A536-C55369206416 (simzone_d) VM RESERVATIONS|1                             
|713BD5E0-ED22-5D4E-8F20-3D89CCAAC261 (simzone_c) VM RESERVATIONS|5                             
|8DFE4032-A15D-5738-9009-1509A57A849C (simzone_b) VM RESERVATIONS|6                             

