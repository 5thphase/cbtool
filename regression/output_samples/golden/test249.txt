 status: Sending a termination request for vm_1 (cloud-assigned uuid 4B8BB9C5-E10F-5BC4-BE6A-450EC81549B5)....
VM object FFBD485E-62F9-55ED-AD0E-F9751F318A3E (named "vm_1") was sucessfully detached from this experiment.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_3          |tinyvm              |pico32      |142.51.103.232  |simhostb1                  |SUT           |private    |none    |none       |1E717DC1-DAF2-5D0C-9D2C-0818F7890FB1
|vm_6          |tinyvm              |pico32      |22.155.58.249   |simhostb14                 |SUT           |private    |none    |none       |7AD069C4-0225-5AA5-8390-B031386648D6
|vm_7          |tinyvm              |pico32      |168.218.237.166 |simhostc9                  |SUT           |private    |none    |none       |7F54B7DE-E36E-5EB6-AA44-460E0B7B8233
|vm_8          |tinyvm              |pico32      |63.244.87.158   |simhostc8                  |SUT           |private    |none    |none       |AF139952-8CA9-5230-93D9-1EF729E2EF96
|vm_9          |db2                 |platinum64  |98.62.76.227    |simhostd5                  |SUT           |private    |none    |none       |CC7FBEBB-72C2-57BD-ABF6-6A5007E4E595
|vm_10         |netclient           |iron32      |138.234.53.105  |simhosta2                  |SUT           |private    |none    |none       |DE243271-F277-5B8F-B64E-CB666AF9E6F7
|vm_11         |fen_hpc             |platinum64  |203.180.229.62  |simhostb1                  |LG            |private    |none    |none       |D6D4C364-AD63-5207-A605-E42A5B37AFBE
|vm_12         |tinyvm              |iron32      |97.180.244.228  |simhostc12                 |SUT           |private    |none    |none       |25E95148-2C01-5B50-97CF-E7CC8CCCE741
|vm_13         |tinyvm              |iron32      |205.1.62.114    |simhostc2                  |SUT           |private    |none    |none       |E876284D-E22B-53DB-83D3-4E5FC2D73BEA
|vm_14         |predictablevm       |iron32      |1.2.3.4         |simhostc2                  |SUT           |private    |none    |none       |8591AE45-C644-5390-A381-C74C64CF5718
|vm_15         |tinyvm              |pico32      |102.197.149.180 |simhostd3                  |SUT           |private    |none    |none       |6A23D608-93B9-5A1B-A790-1278D108987F
|vm_16         |tinyvm              |pico32      |193.171.141.24  |simhostb10                 |SUT           |private    |none    |none       |FE90B68A-589D-5478-8C27-339C9A03A5E5
|vm_17         |tinyvm              |pico32      |121.178.74.193  |simhostb9                  |SUT           |private    |none    |none       |D01E40CD-D603-5821-8D13-FB22AC5B74F2
|vm_18         |tinyvm              |pico32      |211.215.121.136 |simhostc13                 |SUT           |private    |none    |none       |9A82336F-A0DA-5D14-A9AD-638EB2C59F82
|vm_19         |tinyvm              |pico32      |55.191.167.44   |simhosta6                  |SUT           |private    |none    |none       |C0E1A7B0-544F-5334-A909-58919D91C505
|vm_20         |tinyvm              |pico32      |161.142.49.47   |simhostb14                 |SUT           |private    |none    |none       |5809E047-1BC0-5E59-996B-2505B8ABDBB1
|vm_21         |tinyvm              |pico32      |105.28.54.111   |simhosta1                  |SUT           |private    |none    |none       |5728A65F-846E-5752-987F-E0E63C5C084E
|vm_22         |tinyvm              |pico32      |99.157.185.92   |simhostc5                  |SUT           |private    |none    |none       |958C010A-3877-528E-96C7-70B89AFB0223

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |19.48M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |14435154                      
|Total Connections Received                           |614607                        
|Total Commands Processed                             |58392909                      
|Number of Keys                                       |694                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |27                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |18                            
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |18                            
|VMs ISSUED                                           |22                            
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |4                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |0                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |234                           

 ---------------- VMC-WIDE COUNTERS ----------------
|4D841AFA-3629-5865-B277-3C9B87B46AF4 (simzone_d) VM RESERVATIONS|1                             
|B07D97D7-6259-50D3-8A5D-2F6387957881 (simzone_b) VM RESERVATIONS|6                             
|73396447-1A1B-5F78-8FD0-C71D15101114 (simzone_a) VM RESERVATIONS|7                             
|FB6EC147-6C78-5AE6-BD5F-8F8D6A60F5BC (simzone_c) VM RESERVATIONS|4                             

